app/assets/javascripts/admin/controllers/admin-backups-index.js.es6

```diff
@@ -10,11 +10,11 @@ export default Ember.ArrayController.extend({
 
   restoreTitle: function() {
```

Comment:
If you're changing what a property returns (key instead of translation) I think you should rename the CP to reflect it. This is no longer the `restoreTitle` but `restoreTitleKey`